### ISSUE TITLE
Show some top players' games for green hands to learn 

### ISSUE THREAD

**Issue by szzx110** [tobymao/18xx#11235]
REASON:
Only a 1889 toturial is not enough , many games such as 1846 or Rolling stock stars have clear difference from shikoku 1889. 
And, 18xx.games can be a convient system for totur/commentator, because audience can make a self-deduction easily.(If in video,the audience have to set-up for 10mins ,using their boardgame)

METHOD:
Showing some top player's games can be useful.If you have more time, you can bring a rules and platform for totur/commentator, then allow them generate contents(UGC)in the games.
Pick from the finished game , in which the players all have high ELO(only about the single type). (not including games that have just been released)
When players choose to search by game(such as 1846) in the home page(18xx.games) ,you can put them at a prominent position

PROBLEMS: (If the method is carried out) 
Some players may mot want to show themselves/be enaluated.  
You can easily replace all players ID to such as "Player 1","Player 2"......
Only show the players who choose to show the individual stats in their profile, is also OK.

Language border may have a little negative influence for UGC.
no need to fix it when lack of time 

The needs of server will certainly increase depend on degree of changing.
Beyond my capabilities, can't evaluate it

**Comment by philcampeau** [comment]
I don't believe this is something that anyone will want to code here. It has a very limited use. If people want examples of play, they can find may on Youtube for a variety of games, full of commentary.
